Makefile

```diff
@@ -91,11 +91,17 @@ lint:
 # Chain Code Downloads
 ###############################################################################
 
+CHAIN_CODE := ./chain-code
+GAIA_REPO := $(CHAIN_CODE)/gaia
+
 get-gaia:
-	@mkdir -p ./chain-code/
-	@git clone --branch $(GAIA_VERSION) --depth=1 https://github.com/cosmos/gaia.git ./chain-code/gaia
+	@mkdir -p $(CHAIN_CODE)/
+	@git clone --branch $(GAIA_REPO) --depth=1 https://github.com/cosmos/gaia.git $(GAIA_REPO)
 
 build-gaia:
```

Comment:
To my knowledge they are only used if people try and run the `dev-env` script in `examples/demo`

If the gaiad binary doesn't exists, it will recommend you run 'make build-gaia'

Reply:
maybe we should replace the demo examples with ibctest examples? Seems fine to keep this for now though. Thanks for fixing!